sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -44,120 +44,123 @@ lint.dependsOn mypyPy38
 apply from: "../common.gradle"
 
 
-
-// PyCoverage Precommit runs test suites that evaluate test coverage and compatibility of
-// particular dependencies. It is exercised on a single Python version.
+toxTask "testPy38CloudCoverage", "py38-cloudcoverage", "${posargs}"
+test.dependsOn "testPy38CloudCoverage"
 project.tasks.register("preCommitPyCoverage") {
       dependsOn = ["testPy38CloudCoverage"]
 }
 
+// Dep Postcommit runs test suites that evaluate compatibility of particular
+// dependencies. It is exercised on a single Python version.
+project.tasks.register("postCommitPyDep") {}
+
 // Create a test task for each supported major version of pyarrow
 toxTask "testPy38pyarrow-3", "py38-pyarrow-3", "${posargs}"
 test.dependsOn "testPy38pyarrow-3"
-preCommitPyCoverage.dependsOn "testPy38pyarrow-3"
+postCommitPyDep.dependsOn "testPy38pyarrow-3"
 
 toxTask "testPy38pyarrow-4", "py38-pyarrow-4", "${posargs}"
 test.dependsOn "testPy38pyarrow-4"
-preCommitPyCoverage.dependsOn "testPy38pyarrow-4"
+postCommitPyDep.dependsOn "testPy38pyarrow-4"
 
 toxTask "testPy38pyarrow-5", "py38-pyarrow-5", "${posargs}"
 test.dependsOn "testPy38pyarrow-5"
-preCommitPyCoverage.dependsOn "testPy38pyarrow-5"
+postCommitPyDep.dependsOn "testPy38pyarrow-5"
 
 toxTask "testPy38pyarrow-6", "py38-pyarrow-6", "${posargs}"
 test.dependsOn "testPy38pyarrow-6"
-preCommitPyCoverage.dependsOn "testPy38pyarrow-6"
+postCommitPyDep.dependsOn "testPy38pyarrow-6"
 
 toxTask "testPy38pyarrow-7", "py38-pyarrow-7", "${posargs}"
 test.dependsOn "testPy38pyarrow-7"
-preCommitPyCoverage.dependsOn "testPy38pyarrow-7"
+postCommitPyDep.dependsOn "testPy38pyarrow-7"
 
 toxTask "testPy38pyarrow-8", "py38-pyarrow-8", "${posargs}"
 test.dependsOn "testPy38pyarrow-8"
-preCommitPyCoverage.dependsOn "testPy38pyarrow-8"
+postCommitPyDep.dependsOn "testPy38pyarrow-8"
```

Comment:
Each of these categories are not strictly dependency tests, they also test some other functionality which is not necessarily tested elsewhere. I agree we don't need the full matrix of all versions in our precommits, but I think it would be good to keep at least 1 version per type that also runs in the precommit. So for example, we could change this line to 2 lines:

```
preCommitPyCoverage.dependsOn "testPy38pyarrow-8"
postCommitPyDep.dependsOn "testPy38pyarrow-8"
```

but leave the rest of the pyarrow versions for the dependency task

Reply:
I thought these tests are also running on usual precommit, with dependency version unrestricted (see comment above). Let me open a draft PR to double check

Reply:
Only pyarrow and pandas tests were executed in other PreCommit (PreCommit dataframe), so these two tests can be fully moved to postcommit; other tests leave at least one version in PreCommit coverage